### ISSUE TITLE
Cannot convert a pytorch nn model to pysyft LargePrecision model

### ISSUE THREAD

**Issue by refactormyself** [OpenMined/PySyft#2800]
I want create a NN using Pytorch and Pysyft with support for Large Precision. There seems to be a bug preventing this.

**To Reproduce**
```
# N is batch size; D_in is input dimension;
# H is hidden dimension; D_out is output dimension.
N, D_in, H, D_out = 64, 1000, 100, 10
model = torch.nn.Sequential(
    torch.nn.Linear(D_in, H),
    torch.nn.ReLU(),
    torch.nn.Linear(H, D_out),
)
large_model = model.fix_prec(internal_type=torch.int16, precision_fractional=128, verbose=True)
 
```
This is the expected way to create large_model according to [this example](https://github.com/OpenMined/PySyft/blob/master/examples/experimental/Large%20Precision%20Tensor.ipynb)
However, this gives the error `Can't call numpy() on Variable that requires grad. Use var.detach().numpy() instead.` 

But then I can't detach the pytorch nn model, I guess that defeats the point of creating the NN.

**I have tried this on Google Colab and pysft installed with `!pip install syft[udacity]`**

**Comment by LaRiffle** [comment]
Hey,
Large Precision is depreciated and will removed in the future,
Can you use _classic_ Fixed Precision for your usecase?

**Comment by refactormyself** [comment]
Oh, I see, that was why #2734 was closed. I was trying to work on it.
Somehow, I usually end up with a "bad" first issue. Maybe there should be a "good second issue" label.
